webservices/common/models.py

```diff
@@ -437,11 +458,22 @@ class CommitteeReportsPresidential(CommitteeReports):
     transfer_to_other_authorized_committee_period = db.Column(db.Integer)
     transfer_to_other_authorized_committee_ytd = db.Column(db.Integer)
 
+    @property
+    # PAC, Party and Presidential records start May 1993
+    def pdf_url(self):
+        if self.report_year is None or self.report_year < 1993:
+            return None
+        else:
```

Comment:
Lines 466-470 can be replaced with `return utils.make_pdf_url(self.beginning_image_number)`, as in other reports classes.